scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/synchronization/SynchronizationWrapper.java

```diff
@@ -0,0 +1,256 @@
+        return internalAPI.remove(originator, taskId, channel, key);
+    }
+
+    @Override
+    public void putAll(String channel, Map<? extends String, ? extends Serializable> m)
+            throws InvalidChannelException, IOException {
+        internalAPI.putAll(originator, taskId, channel, m);
+    }
+
+    @Override
+    public void clear(String channel) throws InvalidChannelException, IOException {
+        internalAPI.clear(originator, taskId, channel);
+    }
+
+    @Override
+    public Set<String> keySet(String channel) throws InvalidChannelException {
+        return internalAPI.keySet(originator, taskId, channel);
+    }
+
+    @Override
+    public Collection<Serializable> values(String channel) throws InvalidChannelException {
+        return internalAPI.values(originator, taskId, channel);
+    }
+
+    @Override
+    public Set<Map.Entry<String, Serializable>> entrySet(String channel) throws InvalidChannelException {
+        return internalAPI.entrySet(originator, taskId, channel);
+    }
+
+    @Override
+    public Serializable getOrDefault(String channel, String key, Serializable defaultValue)
+            throws InvalidChannelException {
+        return internalAPI.getOrDefault(originator, taskId, channel, key, defaultValue);
+    }
+
+    @Override
+    public void forEach(String channel, String action) throws InvalidChannelException, CompilationException {
+        internalAPI.forEach(originator, taskId, channel, action);
+    }
+
+    @Override
+    public void replaceAll(String channel, String function)
+            throws InvalidChannelException, CompilationException, IOException {
+        internalAPI.replaceAll(originator, taskId, channel, function);
+    }
+
+    @Override
+    public Serializable putIfAbsent(String channel, String key, Serializable value)
+            throws InvalidChannelException, IOException {
+        return internalAPI.putIfAbsent(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public boolean remove(String channel, String key, Serializable value) throws InvalidChannelException, IOException {
+        return internalAPI.remove(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public boolean replace(String channel, String key, Serializable oldValue, Serializable newValue)
+            throws InvalidChannelException, IOException {
+        return internalAPI.replace(originator, taskId, channel, key, oldValue, newValue);
+    }
+
+    @Override
+    public Serializable replace(String channel, String key, Serializable value)
+            throws InvalidChannelException, IOException {
+        return internalAPI.replace(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public Serializable computeIfAbsent(String channel, String key, String mappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)